raml-osprey-expressjs/pokemon_api/index.js

```diff
@@ -0,0 +1,107 @@
+var express = require('express')
```

Comment:
let's make sure to write this file in modern ES6 style, e.g. using `const` for requires and immutable variables like PORT.

makes sense?

Reply:
Yes. Fixed. Now using `const` for everything that is not reassigned and `let` in loops.